### PR TITLE
add doc for operators

### PR DESCRIPTION
**Pull request by XuHuaiyu**
PTAL @shenli

### CODE REVIEW

File: sql/functions-and-operators/operators.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+| [`>>`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_right-shift) | Right shift                              |
+| [`RLIKE`](https://dev.mysql.com/doc/refman/5.7/en/regexp.html#operator_regexp) | Synonym for REGEXP                       |
+| [`SOUNDS LIKE`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#operator_sounds-like) | Compare sounds                           |
+| [`*`](https://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html#operator_times) | Multiplication operator                  |
+| [`-`](https://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html#operator_unary-minus) | Change the sign of the argument          |
+| [`XOR`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_xor) | Logical XOR                              |
+
+## Operator Precedence
+
+Operator precedences are shown in the following list, from highest precedence to the lowest. Operators that are shown together on a line have the same precedence.
```
Comment (shenli) [inline sql/functions-and-operators/operators.md]: Operators that -> Operators these

Comment (shenli) [inline sql/functions-and-operators/operators.md]: on a line -> in the same line

File: sql/functions-and-operators/operators.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+For details, see [here](https://dev.mysql.com/doc/refman/5.7/en/operator-precedence.html).
+
+## Comparison Functions and Operators
+
+| Name                                     | Description                              |
+| ---------------------------------------- | ---------------------------------------- |
+| [`BETWEEN ... AND ...`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_between) | Check whether a value is within a range of values |
+| [`COALESCE()`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#function_coalesce) | Return the first non-NULL argument       |
+| [`=`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_equal) | Equal operator                           |
```
Comment (shenli) [inline sql/functions-and-operators/operators.md]: There are many operators already described in the previous table. Why?

Comment (XuHuaiyu) [reply sql/functions-and-operators/operators.md]: To keep the same format with the MySQL manual.

**Comment by XuHuaiyu** [comment]
PTAL @sunhao2017 @lilin90

File: sql/operators.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+- [Operator Precedence](#operator-precedence)
+- [Comparison Functions and Operators](#comparison-functions-and-operators)
+- [Logical Operators](#comparison-functions-and-operators)
+- [Assignment Operators](#comparison-functions-and-operators)
+
+| Name | Description |
+| ---------------------------------------- | ---------------------------------------- |
+| [`AND`, `&&`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_and) | Logical AND |
+| [`=`](https://dev.mysql.com/doc/refman/5.7/en/assignment-operators.html#operator_assign-equal) | Assign a value (as part of a [`SET`](https://dev.mysql.com/doc/refman/5.7/en/set-variable.html) statement, or as part of the `SET` clause in an[`UPDATE`](https://dev.mysql.com/doc/refman/5.7/en/update.html) statement) |
```
Comment (lilin90) [inline sql/operators.md]: Lack a space before `UPDATE`

**Comment by XuHuaiyu** [comment]
PTAL @lilin90

File: sql/operators.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+| [`>`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_greater-than) | Greater than operator |
+| [`>=`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_greater-than-or-equal) | Greater than or equal operator |
+| [`IS`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_is) | Test a value against a boolean |
+| [`IS NOT`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_is-not) | Test a value against a boolean |
+| [`IS NOT NULL`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_is-not-null) | NOT NULL value test |
+| [`IS NULL`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_is-null) | NULL value test |
+| [`->`](https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#operator_json-column-path) | Return value from JSON column after evaluating path; equivalent to JSON_EXTRACT(). |
+| [`->>`](https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#operator_json-inline-path) | Return value from JSON column after evaluating path and unquoting the result; equivalent to JSON_UNQUOTE(JSON_EXTRACT()). |
```
Comment (lilin90) [inline sql/operators.md]: 1) Remove the period "." in the end of the two sentences, to keep consistent with other rows. 
2) `JSON_EXTRACT()` and `JSON_UNQUOTE(JSON_EXTRACT())` would be better.

File: sql/operators.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
+# Operators
+
+- [Operator Precedence](#operator-precedence)
+- [Comparison Functions and Operators](#comparison-functions-and-operators)
+- [Logical Operators](#comparison-functions-and-operators)
```
Comment (lilin90) [inline sql/operators.md]: wrong link here

File: sql/operators.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+- [Assignment Operators](#comparison-functions-and-operators)
```
Comment (lilin90) [inline sql/operators.md]: Another wrong link

**Comment by XuHuaiyu** [comment]
PTAL @lilin90

**Pull request merged**